chef/cookbooks/keystone/attributes/default.rb

```diff
@@ -40,6 +40,7 @@
 default[:keystone][:api][:admin_host] = "0.0.0.0"
 default[:keystone][:api][:api_host] = "0.0.0.0"
 default[:keystone][:api][:version] = "v2.0"
+default[:keystone][:api][:version_multidomain] = "v3"
```

Comment:
I agree.

I did this thing to isolate the Horizon / multidomain problem with the rest of the system.  If I change all the cloud to use V3 here I was expecting complaints.